BundID.xcodeproj/project.pbxproj

```diff
@@ -74,7 +74,6 @@
 		E1BC3D8128B7E73B007A7B39 /* Licenses in Resources */ = {isa = PBXBuildFile; fileRef = E1BC3D7F28B7E73B007A7B39 /* Licenses */; };
 		E1BC3D8228B7E73B007A7B39 /* Licenses.plist in Resources */ = {isa = PBXBuildFile; fileRef = E1BC3D8028B7E73B007A7B39 /* Licenses.plist */; };
 		E1BC3D8628B7E919007A7B39 /* LicensePlistViewController in Frameworks */ = {isa = PBXBuildFile; productRef = E1BC3D8528B7E919007A7B39 /* LicensePlistViewController */; };
-		E1BD6C792847A2E5003048E4 /* SetupDone.swift in Sources */ = {isa = PBXBuildFile; fileRef = E1BD6C782847A2E5003048E4 /* SetupDone.swift */; };
```

Comment:
Was ist hier mit dem SetupDone screen passiert, weißt du das?

Reply:
Kam wohl durch einen (kaputten) merge, in Zeile 12 ist die Datei nochmal.
Hab da übrigens nichts manuell gelöscht. Das was Xcode automatisch beim deployment target ändern